tests/ha/multi_pod_restart/test_multi_server_pod_restart.py

```diff
@@ -0,0 +1,163 @@
+        self.random_time = int(time.time())
+        self.restore_pod = False
+        self.s3_clean = dict()
+        self.s3acc_name = f"ha_s3acc_{int(perf_counter_ns())}"
+        self.s3acc_email = f"{self.s3acc_name}@seagate.com"
+        LOGGER.info("Precondition: Verify cluster is up and running and all pods are online.")
+        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Precondition: Verified cluster is up and running and all pods are online.")
+        LOGGER.info("Get the value for number pods that can go down for cluster")
+        resp = self.ha_obj.calculate_multi_value(self.csm_obj, len(self.node_worker_list))
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Maximum number of pod that can go down is: %s", resp[1])
+        self.qvalue = resp[1]
+        LOGGER.info("Getting K value for the cluster")
+        resp = self.csm_obj.get_sns_value()
+        if not resp:
+            assert_utils.assert_true(False, "Could not retrieve SNS values of cluster")
+        LOGGER.info("K value for the cluster is: %s", resp[1])
+        self.kvalue = resp[1]
+        self.nvalue = len(self.node_worker_list)
+        LOGGER.info("N value for the given cluster is: %s", self.nvalue)
+        LOGGER.info("COMPLETED: Setup operations. ")
+
+    def teardown_method(self):
+        """
+        This function will be invoked after each test function in the module.
+        """
+        if self.s3_clean:
+            LOGGER.info("Cleanup: Cleaning created s3 accounts and buckets.")
+            resp = self.ha_obj.delete_s3_acc_buckets_objects(self.s3_clean)
+            assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("STARTED: Teardown Operations.")
+        if self.restore_pod:
+            resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                           restore_method=self.restore_method,
+                                           restore_params={"deployment_name": self.deployment_name,
+                                                           "deployment_backup":
+                                                               self.deployment_backup})
+            LOGGER.debug("Response: %s", resp)
+            assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+            LOGGER.info("Successfully restored pod by %s way", self.restore_method)
+        LOGGER.info("Cleanup: Check cluster status and start it if not up.")
+        resp = self.ha_obj.check_cluster_status(self.node_master_list[0])
+        if not resp[0]:
+            resp = self.ha_obj.restart_cluster(self.node_master_list[0])
+            assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Done: Teardown completed.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45588")
+    def test_read_after_server_pods_restart(self):
```

Comment:
can we add multi in test case name?
At both test cases name

Reply:
yes